website/content/v1.11/reference/cli.md

```diff
@@ -134,87 +141,117 @@ talosctl cluster create [flags]
+      --with-siderolink true                     enables the use of siderolink agent as configuration apply mechanism. true or `wireguard` enables the agent, `tunnel` enables the agent with grpc tunneling (default none)
+      --with-tpm1_2                              enable TPM 1.2 emulation support using swtpm
+      --with-tpm2                                enable TPM 2.0 emulation support using swtpm
+      --with-uefi                                enable UEFI on x86_64 architecture (default true)
+      --with-uuid-hostnames                      use machine UUIDs as default hostnames
+      --workers int                              the number of workers to create (default 1)
+```
+
+### Options inherited from parent commands
+
+```
+      --name string          the name of the cluster (default "talos-default")
+      --provisioner string   Talos cluster provisioner to use (default "docker")
+      --state string         directory path to store cluster state (default "/home/user/.talos/clusters")
+```
+
+### SEE ALSO
+
+* [talosctl cluster](#talosctl-cluster)	 - A collection of commands for managing local docker-based or QEMU-based clusters
+
+## talosctl cluster create-docker
+
+Create a local Docker based kubernetes cluster
+
+```
+talosctl cluster create-docker [flags]
+```
+
+### Options
+
+```
+      --config-patch stringArray                 patch generated machineconfigs (applied to all node types), use @file to read a patch from file
+      --config-patch-controlplanes stringArray   patch generated machineconfigs (applied to 'init' and 'controlplane' types)
+      --config-patch-workers stringArray         patch generated machineconfigs (applied to 'worker' type)
+      --cpus-controlplanes string                the share of CPUs as fraction (each control plane/VM) (default "2.0")
+      --cpus-workers string                      the share of CPUs as fraction (each worker/VM) (default "2.0")
+      --disable-ipv6                             skip enabling IPv6 in containers
+  -p, --exposed-ports string                     Comma-separated list of ports/protocols to expose on init node. Ex -p <hostPort>:<containerPort>/<protocol (tcp or udp)>
+  -h, --help                                     help for create-docker
+      --host-ip string                           Host IP to forward exposed ports to (default "0.0.0.0")
+      --kubernetes-version string                desired kubernetes version to run (default "1.34.0-beta.0")
+      --memory-controlplanes int                 the limit on memory usage in MB (each control plane/VM) (default 2048)
+      --memory-workers int                       the limit on memory usage in MB (each worker/VM) (default 2048)
+      --mount mount                              attach a mount to the container
+      --subnet string                            Docker network subnet CIDR (default "10.5.0.0/24")
+      --talos-version string                     the desired Talos version (default "latest")
```

Comment:
we should drop/hide this? we should drive version from the `--image` ? ( I don't see it?)

Reply:
Added back the image flag which is by default derived from the talos version but can be overridden optionally